docs/DeveloperGuide.md

```diff
@@ -720,40 +720,212 @@ testers are expected to do more *exploratory* testing.
 
-1. _{ more test cases …​ }_
-
 ### Deleting a person
 
 1. Deleting a person while all persons are being shown
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+    1. Prerequisites: List all persons using the `listPersons` command. Multiple persons in the list.
+
+    2. Test case: `deletePerson 1`<br>
+       Expected: First person is deleted from the person list. Details of the deleted person shown in the result display.
+
+    3. Test case: `deletePerson 0`<br>
+       Expected: No person is deleted. Error message is shown in the result display.
+
+    4. Other incorrect deletePerson commands to try: `deletePerson`, `deletePerson x`, `...` (where x is larger than the person list size)<br>
+       Expected: Similar to previous.
+
+2. Deleting a person while some persons are being shown
+
+    1. Prerequisites: List only some persons using the `find` command and specify certain keywords.
+
+    2. Test case: `deletePerson 1`<br>
+       Expected: First person is deleted from the current displayed person list. Details of the deleted person shown in the result display.
+
+    3. Test case: `deletePerson 0`<br>
+       Expected: No person is deleted from the current displayed person list. Error message is shown in the result display.
+
+    4. Other incorrect deletePerson commands to try: `deletePerson`, `deletePerson x`, `...` (where x is larger than the person list size)<br>
+       Expected: Similar to previous.
+
+### Adding a person
+
+1. Adding a person
+
+    1. Prerequisites: The name of the newly added person should not exist in any of the current persons stored in the application.
+
+    2. Test case: `addPerson n/person a/street e/person@gmail.com p/12345678 d/01/01/2000 g/m`<br>
+       Expected: A person is added to the end of the person list. Details of the added person shown in the result display.
+
+    3. Test case: `addPerson n/person`<br>
+       Expected: No person is added. Error message is shown in the result display.
+
+    4. Other incorrect addPerson commands to try: `addPerson 1`, `addPerson n/person a/street e/person@gmail.com p/12345678 d/01.01.2000 g/m`, `...`
+       (where the date of birth is in the wrong format)<br>
+       Expected: Similar to previous.
+
+### Listing Events by Event Date
+1. Listing Events by their Event Titles
+    1. Prerequisites: Multiple events in the event list.
+
+    2. Test case: `listEvents s/d`<br>
+       Expected: All events are listed and sorted by their dates from oldest event date to newest event date.
+       A success message is shown in the result display. The top event has the oldest event date.
+
+    3. Test case: `listEvents 123`<br>
+       Expected: Current event list remains the same and no events are sorted. Error message is shown in the result display.
+
+    4. Other incorrect listEvent commands to try: `LISTEVENTS s/d`, `listEvents s/d e`, `listEvents s/n`<br>
+       Expected: Similar to second test case.
```

Comment:
I think the way to refer to test case it's a bit confusing, since the second test case has the index number 3, so maybe you can use this indexing: 
```
* Prerequisites
* test cases
  1. sth
  2. sth
  3. sth
* sth
```